packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -157,18 +163,8 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   };
   /* @conditional-compile-remove(call-readiness) */
   const networkErrors = errorBarProps.activeErrorMessages.filter((message) => message.type === 'callNetworkQualityLow');
-
-  /* @conditional-compile-remove(call-readiness) */
-  const [isDrawerShowing, setIsDrawerShowing] = useState(true);
-  /* @conditional-compile-remove(call-readiness) */
-  const onLightDismissTriggered = (): void => {
-    // do nothing here
-    // only way to dismiss this drawer is clicking on allow access which will leads to device permission prompt
-  };
-
   /* @conditional-compile-remove(call-readiness) */
-  const [isModalShowing, setIsModalShowing] = useState(false);
-
+  const [isModalShowing, setIsModalShowing] = useState(true);
```

Comment:
nit: specify what "modal" is here
```suggestion
  const [isAskPermissionsModalShowing, setIsAskPermissionsModalShowing] = useState(true);
```

Reply:
made changes and pushed to PR https://github.com/Azure/communication-ui-library/pull/2529